### PR TITLE
Add Java 21 jre ubi9 minimal images

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

File: releases/latest/beta/Dockerfile.ubi.openjdk21 [hunk 1]
```diff
@@ -43,8 +44,8 @@ LABEL org.opencontainers.image.authors="Leo Christy Jesuraj, Melissa Lee, Thomas
       vendor="Open Liberty" \
       name="Open Liberty Beta" \
       version="$LIBERTY_VERSION" \
-      summary="Image for Open Liberty Beta with IBM Semeru Runtime Open Edition OpenJDK 17 with OpenJ9 and UBI 8" \
-      description="This image contains the Open Liberty beta runtime with IBM Semeru Runtime Open Edition OpenJDK 17 with OpenJ9 and Red Hat UBI 8 as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image"
+      summary="Image for Open Liberty Beta with IBM Semeru Runtime Open Edition OpenJDK 17 with OpenJ9 and UBI 9 minimal" \
```
Comment (leochr) [inline releases/latest/beta/Dockerfile.ubi.openjdk21:47]: OpenJDK 17 --> OpenJDK 21

File: releases/latest/full/Dockerfile.ubi.openjdk21 [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    && adduser -u 1001 -r -g root -s /usr/sbin/nologin default \
+    && chown -R 1001:0 /opt/ol/wlp \
+    && chmod -R g+rw /opt/ol/wlp
+
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal
+
+USER root
+
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_BUILD_LABEL=replace_with_gm_driver_label
```
Comment (leochr) [inline releases/latest/full/Dockerfile.ubi.openjdk21:32]: It should be `{replace_with_gm_driver_label}` (with curly brackets)

File: releases/latest/kernel-slim/Dockerfile.ubi.openjdk21 [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    && adduser -u 1001 -r -g root -s /usr/sbin/nologin default \
+    && chown -R 1001:0 /opt/ol/wlp \
+    && chmod -R g+rw /opt/ol/wlp
+
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal
+
+USER root
+
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_BUILD_LABEL=replace_with_gm_driver_label
```
Comment (leochr) [inline releases/latest/kernel-slim/Dockerfile.ubi.openjdk21:32]: It should be {replace_with_gm_driver_label} (with curly brackets)

**Review by leochr — COMMENTED** [review]
@tjwatson Thanks for the PR. Looks good. Just a few minor comments.

File: releases/latest/kernel-slim/Dockerfile.ubi.openjdk21 (hunk 3 above)
Comment (tjwatson) [reply releases/latest/kernel-slim/Dockerfile.ubi.openjdk21]: Done

File: releases/latest/full/Dockerfile.ubi.openjdk21 (hunk 2 above)
Comment (tjwatson) [reply releases/latest/full/Dockerfile.ubi.openjdk21]: Done

**Comment by leochr** [comment]
@tjwatson Thanks for the quick updates. Now the Java 21 Dockerfiles for _full_ and _kernel-slim_ under `latest` can be copied over as is to the `24.0.0.1` directory.

**Comment by tjwatson** [comment]
> @tjwatson Thanks for the quick updates. Now the Java 21 Dockerfiles for _full_ and _kernel-slim_ under `latest` can be copied over as is to the `24.0.0.1` directory.

Done

**Review by leochr — DISMISSED** [review]

**Review by mbroz2 — DISMISSED** [review]

**Review by leochr — APPROVED** [review]

**Pull request merged**